src/ng-select/lib/ng-select.component.html

```diff
@@ -1,4 +1,4 @@
-<div (mousedown)="handleMousedown($event)" [class.ng-has-value]="hasValue" class="ng-select-container">
+<div (click)="handleClick($event)" (contextmenu)="handleContextmenu($event)" [class.ng-has-value]="hasValue" class="ng-select-container">
```

Comment:
why have you changed to `click` instead of `mousedown`?

Reply:
Hi @varnastadeus, 
Because, AFAIK, it is not possible to safely determine from a MouseEvent whether it was a left or a right click. According to the documentation of https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button, button 2 is **usually** the right button, but that is not always the case. 
So that is why I changed it to `click` and `contextmenu`.